src/index.html

```diff
@@ -6,22 +6,339 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Catalog</title>
+    <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
     <link
       rel="preconnect"
       href="https://fonts.gstatic.com"
+      crossorigin="true"
     />
     <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
       rel="stylesheet"
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="logo-container"
+        href="#"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
```

Comment:
```suggestion
    <header class="header">
      <a
        class="logo-container"
        href="#"
      >
        <img
          src="./images/logo.png"
          alt="logo"
        />
      </a>
      
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              class="nav__link is-active"
              href="#"
            >
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a
              class="nav__link"
              href="#"
            >
              Samsung
            </a>
          </li>
```

It is a good idea to add blank line between multiline siblings. Fix everywhere